examples/cifar/train_cifar_custom_loop.py

```diff
@@ -37,7 +38,7 @@ def main():
     parser.add_argument('--test', action='store_true',
                         help='Use tiny datasets for quick tests')
     parser.add_argument('--resume', '-r', default='',
```

Comment:
Let's default to `None`, and in that case the `default` argument can be skipped altogether.